src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -334,6 +334,31 @@ public static void ACos_Advanced(double real, double imaginary, double expectedR
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        public static IEnumerable<object[]> ACos_Legacy_TestData ()
+        {
+            // Boundary values
+            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
+            yield return new object[] { double.MinValue, 0, double.NaN, double.NaN };
+
+            // Invalid values
+            foreach (double invalidReal in s_invalidDoubleValues) {
```

Comment:
I did open an issue (https://github.com/dotnet/corefx/issues/16972) to track adding the appropriate style options to our editrconfig file (and for CoreCLR https://github.com/dotnet/coreclr/issues/10089).